css/main.css

```diff
@@ -1 +1,24 @@
-exit
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
+body {
+  margin: auto;
+  max-width: 1366px;
+}
+
+@font-face {
+  font-family: font_main;
+  src: url("../font/UTM BryantLG.ttf");
+}
+
+.font-main {
+  font-family: font_main;
+  font-style: normal;
+  letter-spacing: -0.05em;
+  color: #ffffff;
+}
```

Comment:
thiếu end of line. Github báo này em 
![image](https://user-images.githubusercontent.com/2556644/87914191-7ed6c480-ca9a-11ea-8b9f-c8acdce3429a.png)